frontend/src/metabase/search/util/filter-types.ts

```diff
@@ -0,0 +1,23 @@
+import { FC } from "react";
+import { SearchModelType } from "metabase-types/api";
+
+export const SearchFilterKeys = {
```

Comment:
I'm using this object because we'll want to extract keys from the search URL, so having these keys in an actual object (instead of a string literal type) is crucial for me. It also allows us to reference a SearchFilterKey without having to use the hardcoded string, which should reduce any typo weirdness. (_I know we have help from the IDE but I think it would be better to not rely on that_